### PR TITLE
Update Input.py

### PR DESCRIPTION
**Pull request by HighnessAtharva**
Working Solution Added.

### CODE REVIEW

File: BuiltIns/Input.py [hunk 1]
```diff
@@ -4,7 +4,15 @@
...
 Created   : 15 July 2016
+Updated   : 3 April 2021
 Problem   : https://www.hackerrank.com/challenges/input/problem
 '''
-x,k=list(map(int,raw_input().split()))
-print(input() == k)
+
+if __name__ == "__main__":
+    x, k = map(int, input().strip().split())
+    string = input().strip()
```
Comment (arsho) [inline BuiltIns/Input.py:13]: We may simply this portion: 

```
if __name__ == "__main__":
    x, k = map(int, input().strip().split())
    equation = input().strip()
    print(eval(equation) == k)
```

**Review by arsho — CHANGES_REQUESTED** [review]
@HighnessAtharva ,

Thank you for the update. Can you accommodate the review comment?

Cheers!

File: BuiltIns/Input.py (hunk 1 above)
Comment (HighnessAtharva) [reply BuiltIns/Input.py]: That certainly looks better. Done!

**Review by arsho — APPROVED** [review]
The change request is accommodated perfectly. Thank you.

**Pull request merged**